src/policy/auth_logic/BUILD

```diff
@@ -17,6 +17,7 @@
 load("//build_defs:antlr.bzl", "antlr4_cc_combined")
 
 package(
+    default_visibility = ["//visibility:public"],
```

Comment:
A `public` default visibility is generally discouraged. Try to enumerate the directories where you need this. If you find whole subtrees of the repo need this, feel free to use the `:__subpackages__` target.

Reply:
Looks like making this change does not resolve the issue